scripts/decrypt.sh

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env bash
-# This script decrypts our test database 
-# WARNING: Edit decrypt.template.mustache not decrypt.sh which is a generated file
+# This script decrypts our test database
 # Bash3 Boilerplate. Copyright (c) 2014, kvz.io
+# You will need the environment variables CIRCLE_CI_KEY and CIRCLE_CI_IV
 
 set -o errexit
 set -o pipefail
 set -o nounset
 set -o xtrace
 
-openssl aes-256-cbc -K $encrypted_f3e772af0e32_key -iv $encrypted_f3e772af0e32_iv -in secrets.tar.enc -out secrets.tar -d
+: "$CIRCLE_CI_KEY"
+: "$CIRCLE_CI_IV"
+
+openssl aes-256-cbc -d -in circle_ci_test_data.zip.enc -k "$CIRCLE_CI_KEY" -iv "$CIRCLE_CI_IV" -out secrets.tar
 tar xvf secrets.tar
```

Comment:
For my understanding, the other confidential files like config_file, confidential1_1.5.0 are being expanded out to their final locations?

Reply:
everything will go to the right location auto-magically